app/capabilities/capabilities.tsx

```diff
@@ -42,6 +43,7 @@ export class Capabilities {
     this.userOwnedExecutors = Boolean(config.user_owned_executors_enabled);
     this.executorKeyCreation = Boolean(config.executor_key_creation_enabled);
     this.code = Boolean(config.code_editor_enabled);
+    this.chunkedEventLogs = Boolean(config.chunked_event_logs_enabled);
```

Comment:
Do we need this? Can we just use the presence of a lastChunkID? 

If the server sets that - we do chunking, otherwise we display the consoleBuffer.

Reply:
Agree -- @tempoz if you're planning on changing the `last_chunk_id` field of the `Invocation` to just be a bool like `chunked_event_logs_enabled`, mind also having that bool be conditional on the server side flag?

Reply:
I fixed that in my PR now.